tests/lib/eslint.js

```diff
@@ -3116,15 +3112,50 @@ describe("eslint", function() {
             assert.equal(messages[2].column, 18);
         });
 
-        it("should properly parse let declaration when passed ecmaVersion", function() {
+        describe("ecmaVersion", function() {
+            describe("it should properly parse let declaration when", function() {
+                it("the ECMASCript version number is 6", function() {
+                    const messages = eslint.verify("let x = 5;", {
+                        parserOptions: {
+                            ecmaVersion: 6
+                        }
+                    });
 
-            const messages = eslint.verify("let x = 5;", {
-                parserOptions: {
-                    ecmaVersion: 6
-                }
-            }, filename);
+                    assert.equal(messages.length, 0);
+                });
 
-            assert.equal(messages.length, 0);
+                it("the ECMASCript version number is 2015", function() {
+                    const messages = eslint.verify("let x = 5;", {
+                        parserOptions: {
+                            ecmaVersion: 2015
+                        }
+                    });
+
+                    assert.equal(messages.length, 0);
+                });
+            });
+
+            describe("should properly parse exponentiation operator when", function() {
+                it("the ECMASCript version number is 7", function() {
+                    const messages = eslint.verify("x ** y;", {
+                        parserOptions: {
+                            ecmaVersion: 7
+                        }
+                    });
+
+                    assert.equal(messages.length, 0);
+                });
+
+                it("the ECMASCript version number is 2016", function() {
+                    const messages = eslint.verify("x ** y;", {
+                        parserOptions: {
+                            ecmaVersion: 2016
+                        }
+                    });
+
+                    assert.equal(messages.length, 0);
+                });
+            });
```

Comment:
Would it be worth it to add a test ensuring that it fails to parse the exponentiation operator when the version is 2015? If not, LGTM!